agenta-web/cypress/utils/helpers/createVariantsAndTestsets.ts

```diff
@@ -0,0 +1,81 @@
+import {randString} from "../../../src/lib/helpers/utils"
+
+export let app_id
+export function createVariantsAndTestsets() {
```

Comment:
I'd suggest to define this as a reusable Cypress command instead of a helper function. See `clickLinkAndWait` command as an example in `cypress/support/commands/utils.ts` file. Also, please move this to the same file.